app/src/NativeBase/Components/Forms/ResponseModal.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file Modal shows response from api.
+ */
+
+import React, { FC } from 'react'
+import { Text, Modal, Icon } from 'native-base'
+import { Pressable } from 'react-native'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+
+interface ResponseModalProps {
+  message: string
+  header: string
+  success: boolean
+  isOpen: boolean
+  onClose: (value: boolean) => void
+}
+
+const ResponseModal: FC<ResponseModalProps> = ({
+  message,
+  header,
+  success,
+  isOpen,
+  onClose,
+}) => {
+  if (success) {
+    return (
+      <Modal
+        isOpen={isOpen}
+        onClose={onClose}
+        justifyContent="center"
+        size="xl"
+      >
+        <Modal.Content
+          _dark={{ color: 'textDarkMode.100' }}
```

Comment:
I couldn't spot any difference if these `_dark` and `_light` properties are removed on `Modal.Content` (unless it's very subtle?)  If it's already using the default colours here, can remove these?